fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -438,6 +445,7 @@ primaryExpression
     | identifier                                                                               #columnReference
     | base=primaryExpression DOT fieldName=identifier                                          #dereference
     | LEFT_PAREN expression RIGHT_PAREN                                                        #parenthesizedExpression
+    | LEFT_PAREN namedExpression (COMMA namedExpression)+ RIGHT_PAREN                                    #rowConstructor
```

Comment:
if u use `rowConstructor` as expression. This mean u want to support syntax like this `select (1, 2, 3)`. but u do not process it at all.